### PR TITLE
Update residency.njk

### PR DESCRIPTION
**Pull request by gregknight1**
I looked at this before Christmas and you all gave feedback to get it to roughly this state.

It's since been shown to providers at NASBTT who gave some feedback to my question on 'most SCITTS'

- I suggested that 'Most SCITTs' wasn't helpful so added context around that, including a link so people can check if a provider has a sponsor licence. 
- Made the GiT link consistent with the work that's been done on these.

### CODE REVIEW

**Review by Tomcowan1 — APPROVED** [review]

File: app/views/application/personal-details/residency.njk [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
 {% block primary %}
 
-<p class="govuk-body">To get help with student visas and your immigration status, <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">register with Get Into Teaching</a> . You will be able to speak to an adviser.</p>
-<p class="govuk-body">Most School Centred Initial Teacher Training (SCITT) providers cannot sponsor student visas for trainees on unsalaried courses.</p>
+<p class="govuk-body">To get help with student visas and your immigration status,speak to a <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">Get Into Teaching</a> adviser.</p>
```
Comment (paulrobertlloyd) [inline app/views/application/personal-details/residency.njk:14]:
```suggestion
<p class="govuk-body">To get help with student visas and your immigration status, speak to a <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">Get Into Teaching</a> adviser.</p>
```

@Tomcowan1 Should we be linking to the adviser service here? https://beta-adviser-getintoteaching.education.gov.uk

Comment (frankieroberto) [inline app/views/application/personal-details/residency.njk:14]: Should we avoid using the Brand Name for GIT here, and just use "teacher training adviser" (which seems to be how they describe them on GIT)?

```suggestion
<p class="govuk-body">To get help with student visas and your immigration status,speak to a <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">teacher training adviser</a>.</p>
```

**Review by paulrobertlloyd — APPROVED** [review]
Just one question about where we link to on GIT

**Review by frankieroberto — COMMENTED** [review]
Looks good.

This wasn't part of your changes, but I found the "Find out about student visas, visa routes that do not need a sponsor and visa options for trainees on salaried courses" link quite hard to parse on first read. Possibly partly because the whole thing is underlined.

**Review by paulrobertlloyd — APPROVED** [review]
